### ISSUE TITLE
I get the exit error 1 when i start the game

### ISSUE THREAD

**Issue by emmod-190** [Robocraft999/CreateGoggles#28]
When I start the game I get a text like this:
Game crashed
An unexpected issue occurred and the game has crashed. We're sorry for the inconvenience.
Exit Code: 1

I have the problem on version 1.20.1. How can I fix this problem?

**Comment by Robocraft999** [comment]
Could you provide me a crash-report please?

**Comment by emmod-190** [comment]
> Could you provide me a crash-report please?

I dont get a txt file in the Minecraft crash-reports folder. I dont know why, when I press start the game the forge loading window opens and closes, after it closes I get the text I wrote above. Would a log or other file help you?

**Comment by Robocraft999** [comment]
Hm. Yes, if there is no crash-report the debug.log or if not present the latest.log might help

**Comment by emmod-190** [comment]
> Hm. Yes, if there is no crash-report the debug.log or if not present the latest.log might help

I dont want to throw away the whole file, maybe the part I dont want to throw away may also be important, but for security reasons, Im sharing the part that says error for now.

Error section inside the latest.log and debug.log file:
```
[27Dec2023 00:20:28.769] [main/ERROR] [mixin/]: Critical injection failure: LVT in net/minecraft/client/renderer/entity/ItemRenderer::m_115143_(Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/item/ItemDisplayContext;ZLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;IILnet/minecraft/client/resources/model/BakedModel;)V has incompatible changes at opcode 380 in callback creategoggles.mixins.json:ItemRendererMixin->@Inject::onRender(Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/item/ItemDisplayContext;ZLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;IILnet/minecraft/client/resources/model/BakedModel;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;ZZLjava/util/Iterator;Lnet/minecraft/client/resources/model/BakedModel;Ljava/util/Iterator;Lnet/minecraft/client/renderer/RenderType;Lcom/mojang/blaze3d/vertex/VertexConsumer;)V.
 Expected: [Z, Z, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;, Ljava/util/Iterator;, Lnet/minecraft/client/renderer/RenderType;, Lcom/mojang/blaze3d/vertex/VertexConsumer;]
    Found: [Z, Z, Z, Ljava/lang/Iterable;, Ljava/lang/Iterable;, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;]
Available: [Z, Z, Z, Ljava/lang/Iterable;, Ljava/lang/Iterable;, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;, Ljava/util/Iterator;, Lnet/minecraft/client/renderer/RenderType;, Lcom/mojang/blaze3d/vertex/VertexConsumer;, Lcom/mojang/blaze3d/vertex/PoseStack$Pose;]

```

Both files have the same error. Maybe there is no harm in sharing the whole file, but I am not sure, I would like to keep it just in case :)

**Comment by Robocraft999** [comment]
Nah, thats exactly what I need. What Forge/Fabric, Create and Creategoggles version do you use?

**Comment by emmod-190** [comment]
> Nah, thats exactly what I need. What Forge/Fabric, Create and Creategoggles version do you use?

Forge version: 1.20.1 - 47.2.18
Create version: create-1.20.1-0.5.1.f
Creategoggles version: 1.20.1-0.5.5.e_hf-[FORGE]
Architectury API version: architectury-9.1.12-forge

**Comment by Robocraft999** [comment]
Do you use other mods? Especially something like Optifine? For me, loading the game with only create, creategoggles, architectury and forge in your specified versions works fine for me. This might be the same as #20 if you look at the given logs: 
```
creategoggles.mixins.json:ItemRendererMixin->@Inject::onRender(Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/client/renderer/block/model/ItemTransforms$TransformType;ZLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;IILnet/minecraft/client/resources/model/BakedModel;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;ZZLjava/util/Iterator;Lnet/minecraft/client/resources/model/BakedModel;Ljava/util/Iterator;Lnet/minecraft/client/renderer/RenderType;Lcom/mojang/blaze3d/vertex/VertexConsumer;)V.
 Expected: [Z, Z, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;, Ljava/util/Iterator;, Lnet/minecraft/client/renderer/RenderType;, Lcom/mojang/blaze3d/vertex/VertexConsumer;]
    Found: [Z, Z, Z, Ljava/lang/Iterable;, Ljava/lang/Iterable;, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;]
Available: [Z, Z, Z, Ljava/lang/Iterable;, Ljava/lang/Iterable;, Ljava/util/Iterator;, Lnet/minecraft/client/resources/model/BakedModel;, Ljava/util/Iterator;, Lnet/minecraft/client/renderer/RenderType;, Lcom/mojang/blaze3d/vertex/VertexConsumer;, Lcom/mojang/blaze3d/vertex/PoseStack$Pose;]
```

**Comment by emmod-190** [comment]
I also installed other mods but to make sure that the creategoggles mod was not working I deleted the other mods and left only Create, architectury, creategoggles and Optifine_1.20.1_HD_U_I6. I also deleted Optifine and tried it and this time it worked, so the error is caused by Optifine. I also downloaded Optifine_1.20.1_HD_U_I5 and tried with it and it gave me an error. Im sorry, I should have deleted Optifine and tried it before I wrote here, but I never thought it was because of it...
